doc/source/user_guide/numpy_images.rst

```diff
@@ -136,30 +135,30 @@ the grayscale image above:
 Coordinate conventions
 ----------------------
 
-Because we represent images with numpy arrays, our coordinates must
-match accordingly. Two-dimensional (2D) grayscale images (such as
-`camera` above) are indexed by row and columns (abbreviated to either
-``(row, col)`` or ``(r, c)``), with the lowest element ``(0, 0)`` at
-the top-left corner. In various parts of the library, you will
-also see ``rr`` and ``cc`` refer to lists of row and column
-coordinates. We distinguish this from ``(x, y)``, which commonly denote
-standard Cartesian coordinates, where ``x`` is the horizontal coordinate,
-``y`` the vertical, and the origin is on the bottom left.
-(Matplotlib, for example, uses this convention.)
-
-In the case of color (or multichannel) images, the last dimension
-contains the color information and is denoted ``channel`` or ``ch``.
-
-Finally, for 3D images, such as videos, magnetic resonance imaging
-(MRI) scans, or confocal microscopy, we refer to the leading dimension
+Because ``scikit-image`` represents images using NumPy arrays, the
+coordinate conventions must match. Two-dimensional (2D) grayscale images
+(such as `camera` above) are indexed by rows and columns (abbreviated to
+either ``(row, col)`` or ``(r, c)``), with the lowest element ``(0, 0)``
+at the top-left corner. In various parts of the library, you will
+also see ``rr`` and ``cc`` referring to lists of row and column
```

Comment:
"refer" was correct here. (This is also correct, but sounds a bit funnier to me.)